### PR TITLE
Don't depend on go-ipfs directly

### PR DESCRIPTION
**Pull request by magik6k**
Fixes https://github.com/ipfs/iptb/issues/48
Merge with https://github.com/ipfs/go-ipfs/pull/5277

### CODE REVIEW

File: util/localnode.go [hunk 1]
```diff
@@ -14,11 +14,10 @@ import (
 	"syscall"
 	"time"
 
-	config "github.com/ipfs/go-ipfs/repo/config"
-	serial "github.com/ipfs/go-ipfs/repo/fsrepo/serialize"
-
 	ma "github.com/multiformats/go-multiaddr"
 	manet "github.com/multiformats/go-multiaddr-net"
+	config "gx/ipfs/QmSzU7M24odFR3icDsAK8P8SQoaELzDFv3n3fTPy2grrEL/go-ipfs-config"
```
Comment (Stebalien) [inline util/localnode.go]: Probably shouldn't be checked-in re-written.

**Comment by magik6k** [comment]
Should be good now

**Comment by Stebalien** [comment]
We can merge this even before we merge the go-ipfs change, right?

**Comment by magik6k** [comment]
Yep, it should be fine

**Comment by Stebalien** [comment]
(yolo)

**Pull request merged**